### ISSUE TITLE
Regarding previous issues which are not solved yet from last some months - COEP

### ISSUE THREAD

**Issue by sushilmedhe** [Virtual-Labs/engineers-forum#222]
Hello,
We have created issues for the following major reasons : 
1) Changing logo of the virtual lab IIITH in every experiment at the top of experiment from last some months and We have commented every time for that. But no such response from your side. We need to change the logo with our own COEP logo. We are having much problem while presenting at noodle center. What are the updates about that??

2) We have already mentioned about having problem with the introduction of every lab. As per your requirement we have provided updated description about each lab within time but still it is not updated from last some months. 

3) We have provided all the information about fpga lab and its process and all the stuff which regards to the same experiment. But no one giving a single proper response to us about any of our issue. Refer issue #151 

4) Biomedical and Signal processing Laboratory Lab is not working from COEP institute directory lab list. URL problem is there with the lab. Refer issue #182  

Please take a look on this issues and response us.

**Comment by khushpreet-kaur** [comment]
@sushilmedhe What is the status on this?

**Comment by sushilmedhe** [comment]
@khushpreet-kaur  We need to host some of the labs again which are not reflecting the proper content.

**Comment by sushilmedhe** [comment]
@khushpreet-kaur @sanchita-dixit  for #182 about biomedical and signal processing lab url (point 4 in this issue) refer the following screen shots.

![bio lab](https://cloud.githubusercontent.com/assets/8788705/12974055/0a2a777e-d0d7-11e5-87f9-8ce265167af2.png)
![bio lab 1](https://cloud.githubusercontent.com/assets/8788705/12974058/0cbf764c-d0d7-11e5-99ba-9f28a28a2ac1.png)


The url should be changed when we go through college lab list.

**Comment by sanchita-dixit** [comment]
@sushilmedhe For the first issue mentioned above regarding logo of labs, please create a separate issue if it is not yet resolved. Rest issues have been taken care of.

**Comment by sushilmedhe** [comment]
@sanchita-dixit 

1) The issue regarding the logo will be automatically solved once UI of the lab will be upgraded to new UI of Vlabs.ac.in. 
2) The issue regarding introduction has been resolved in previous updates. for remaining labs it will be reflected with next hosting.
3) The Bio-medical and and Signal processing Lab link problem you have solved before some days.

4)The only problem in this issue is fpga lab which is not working. all above 3 issues are almost solved except fpga. 

I have requested already for re-hosting for labs that must be hosted ASAP. 
3 labs are in queue for hosting are as follow - 
1)Electrical machines 2)Sensor lab 3) Hybrid lab.

kindly look into this.
